executor/distsql.go

```diff
@@ -506,8 +510,10 @@ type IndexLookUpExecutor struct {
 	handleIdx    int
 	tableRequest *tipb.DAGRequest
 	// columns are only required by union scan.
-	columns  []*model.ColumnInfo
-	priority int
+	columns        []*model.ColumnInfo
+	priority       int
+	indexStreaming bool
+	tableStreaming bool
```

Comment:
Do we need to set TableReaderExecutor.streaming's value to `tableStreaming`  in `buildTableReader`?

Reply:
Fixed. @zimulala